electron/main.development.js

```diff
@@ -141,35 +141,42 @@ app.on('ready', async () => {
 
     const tlsConfig = ClientApi.tlsInit(ca);
     let messageCallback, errorCallback = null;
-
-    notify(
-      ca,
-      function handleNotifyMessage(...args) {
-        if (messageCallback) {
-          try {
-            messageCallback(...args);
-          } catch (e) {
-            // The callback might have been released on page refresh etc.
-            messageCallback = null;
+    let isNotiyRunning = false;
```

Comment:
The value assigned to this variable is not really changing as far as I can see. It is defined as `let` so I suppose the intention was to change it, right?

Reply:
Great catch, that's a bug - will fix now